styles.css

```diff
@@ -496,3 +501,104 @@ h2 {
   align-items: center;
   justify-content: space-between;
 }
+
+/* contact form  */
+.contact {
+  display: flex;
+  flex-direction: column;
+  background-image: url("images/contact\ form\ shapes-mobile@2x.png"), url("images/Contact-me.png");
+  background-repeat: no-repeat;
+  background-position: right, center;
+  margin-top: 100px;
+  background-size: auto 73%, 100% 100%;
+  width: 100%;
+  height: 789px;
+}
+
+.contact-deets {
+  font-family: 'Poppins', sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+.contact-deets h2 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #fff;
+}
+
+.contact-deets p {
+  margin-top: -90px;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  margin-top: 30px;
+  text-align: center;
+  height: 100%;
+  justify-content: center;
+  align-items: center;
+}
+
+.form-fields {
+  display: flex;
+  flex-direction: column;
+}
+
+.name {
+  height: 48px;
+  width: 300px;
+  border-radius: 8px;
+  border: 0.01px;
+}
+
+.email {
+  height: 48px;
+  width: 300px;
+  border-radius: 8px;
+  margin-top: 20px;
+  border: 0.01px;
+}
+
+.text {
+  margin-top: 20px;
+  width: 300px;
+  border-radius: 8px;
+  height: 150px;
+}
+
+.submit {
+  margin-top: 20px;
+  color: #396df2;
+  height: 60px;
+  width: 180px;
+  border-radius: 8px;
+  background-color: white;
+  border: 2px solid #6070ff;
+}
```

Comment:
- [ ] Please remove the white space under the submit button. Also, decrease the padding of the button to match the figma design.
![image](https://user-images.githubusercontent.com/91486791/220368467-4e2bb2ca-c1f0-4f86-9cf7-4814966f7d71.png)
figma:
![image](https://user-images.githubusercontent.com/91486791/220367989-6619df50-5fa7-4d3a-b4f1-d74955abe3c4.png)
- [ ] One more thing, kindly add these button interactions in the figma design to make in more enhancing.
![image](https://user-images.githubusercontent.com/91486791/220368271-a0a0a212-4c9b-46ec-9a45-45af4aa0f58a.png)